src/App.tsx

```diff
@@ -22,27 +22,27 @@ function App() {
       {
         title,
         completed: false,
-        key: uuidv4(),
+        id: uuidv4(),
```

Comment:
Now that our interface is being used, all the places we aren't properly shaping our data become clear with eslint errors.